sceptre/scipool/config/develop/sc-product-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -9,6 +9,63 @@ dependencies:
+      Name: 'v1.0.2'
+    - Description: 'adds alb-listener-rule custom resource, enabling connection outside VPN.'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.1/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.1'
+    - Description: 'Remove Jumpcloud Integration.'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.2/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.2'
+    - Description: 'Update Cloudformation hooks'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.3/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.3'
+    - Description: 'Use SynapseTagger'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.7/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.7'
+    - Description: 'Remove VpcName input'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.8/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.8'
+    - Description: 'Update for strides'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.9/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.9'
+    - Description: 'Include params in mapping'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.10/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.10'
+    - Description: 'Refactor apache script'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.12/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.12'
+    - Description: 'Fix policies'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.13/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.13'
+    - Description: 'Restore set_env_var.'
+      Info:
+        LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.1.14/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
+      Name: 'v1.1.14'
+    - Description: 'Synapse Credential passthrough. {{ range(1, 10000) | random }}'
```

Comment:
I don't think this needs to be a random number, and setting it to a random value could be troublesome. Why not add a "tag" to the description? Just append `[LATEST]` to the description and remove the "tag" later when we release a new version? Changing the description string should force an update.